src/Symfony/Component/Console/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * Add `SignalMap` to map signal value to its name
  * Multi-line text in vertical tables is aligned properly
+ * The application can also catch errors with `Application::setCatchErrors(true)`
```

Comment:
It would be nice to describe a bit in which case it is useful, here and on the setter docblock

Reply:
For the PHP Docs, I copied the one from `setCatchExceptions()`. And usually, we don't put description about usages in CHANGELOG, does'nt?